src/main/cpp/sequencercontroller.h

```diff
@@ -62,9 +62,9 @@ class SequencerController
         BulkCacher* getBulkCacher();
         void cacheAudioEventsForMeasure( int aMeasure );
 
-        void setBounceState             ( bool aIsBouncing, int aMaxBuffers, char* aOutputDirectory );
-        void setRecordingState          ( bool aRecording,  int aMaxBuffers, char* aOutputDirectory );
-        void setRecordingFromDeviceState( bool aRecording,  int aMaxBuffers, char* aOutputDirectory );
+        void setBounceState             ( bool aIsBouncing, int aMaxBuffers, char* aOutputFile );
+        void setRecordingState          ( bool aRecording,  int aMaxBuffers, char* aOutputFile );
+        void setRecordingFromDeviceState( bool aRecording,  int aMaxBuffers, char* aOutputFile );
 
```

Comment:
Just specify the desired filename of the final recording and let the engine take care of temporary snippets, etc.